app/javascript/actionsPanel/actionsPanel.js

```diff
@@ -109,7 +109,97 @@ function updateExperienceLevel(currentUserId, articleId, rating, group) {
   );
 }
 
+function toggleSubmitContainer() {
+  document
+    .getElementById('adjustment-reason-container')
+    .classList.toggle('hidden');
+}
+
+function adjustTag(tagBtn) {
+  const reasonForAdjustment = document.getElementById('tag-adjustment-reason')
+    .value;
+  const body = {
+    tag_adjustment: {
+      // TODO: change to tag ID
+      tag_name: tagBtn.dataset.tagName,
+      article_id: tagBtn.dataset.articleId,
+      adjustment_type:
+        tagBtn.dataset.adjustmentType === 'subtract' ? 'removal' : 'addition',
+      reason_for_adjustment: reasonForAdjustment,
+    },
+  };
+
+  request('/tag_adjustments', {
+    method: 'POST',
+    body: JSON.stringify(body),
+  })
+    .then((response) => response.json())
+    .then((json) => {
+      if (json.status === 'Success') {
+        const { tagName } = tagBtn.dataset;
+        tagBtn.remove();
+        toggleSubmitContainer();
```

Comment:
Almost there Andy!

So I added a tag and wrote my reason for it. After submitting, I decided to delete a tag; when the SubmitContainer toggled open again, my last entry was still there.

It's minor, but perhaps we should clear the contents of the SubmitContainer after submission?